ClosedXML_Examples/Sparklines/Sparklines.cs

```diff
@@ -0,0 +1,128 @@
+
+            ////Copy worksheet and ensure the sparkline groups are copied
+            //ws.CopyTo("CopyWorksheet");
+
+            ////Create a worksheet and copy a Sparkline group to it
+            //var wsCopySparklineGroup = workbook.AddWorksheet("CopySparklineGroup");
+
+            //wsCopySparklineGroup.Cell("A1").Value = "10";
+            //wsCopySparklineGroup.Cell("B1").Value = "20";
+            //wsCopySparklineGroup.Cell("C1").Value = "30";
+            //wsCopySparklineGroup.Cell("D1").Value = "40";
+            //wsCopySparklineGroup.Cell("E1").Value = "50";
+
+            //wsCopySparklineGroup.Cell("A2").Value = "50";
+            //wsCopySparklineGroup.Cell("B2").Value = "20";
+            //wsCopySparklineGroup.Cell("C2").Value = "30";
+            //wsCopySparklineGroup.Cell("D2").Value = "10";
+            //wsCopySparklineGroup.Cell("E2").Value = "40";
+
+            //slg.CopyTo(wsCopySparklineGroup);
+
+            // Create a worksheet and delete a SparklineGroup from it
+            //var wsDelete = workbook.AddWorksheet("Delete");
+
+            //wsDelete.Cell("A1").Value = "10";
+            //wsDelete.Cell("B1").Value = "20";
+            //wsDelete.Cell("C1").Value = "30";
+            //wsDelete.Cell("D1").Value = "40";
+            //wsDelete.Cell("E1").Value = "50";
+
+            //wsDelete.Cell("A2").Value = "50";
+            //wsDelete.Cell("B2").Value = "20";
+            //wsDelete.Cell("C2").Value = "30";
+            //wsDelete.Cell("D2").Value = "10";
+            //wsDelete.Cell("E2").Value = "40";
+
+            //var slgDelete = wsDelete.SparklineGroups.Add(wsDelete);
+            //slgDelete.SeriesColor = XLColor.CarrotOrange;
+            //slgDelete.AddSparkline(wsDelete.Cell("F1"), wsDelete.Range("A1:E1").RangeAddress.ToStringRelative(true));
+            //slgDelete.AddSparkline(wsDelete.Cell("F2"), wsDelete.Range("A2:E2").RangeAddress.ToStringRelative(true));
+
+            //wsDelete.SparklineGroups.Remove(slgDelete);
+            
+            // Create a worksheet add a sparkline to a cell, remove the cell
+            var wsRemoveCell = workbook.AddWorksheet("RemoveCell");
+
+            wsRemoveCell.Cell("A1").Value = "10";
+            wsRemoveCell.Cell("B1").Value = "20";
+            wsRemoveCell.Cell("C1").Value = "30";
+            wsRemoveCell.Cell("D1").Value = "40";
+            wsRemoveCell.Cell("E1").Value = "50";
+
+            var slgRemoveCell = wsRemoveCell.SparklineGroups.Add(wsRemoveCell);
+            slgRemoveCell.SeriesColor = XLColor.CarrotOrange;
+            slgRemoveCell.AddSparkline(wsRemoveCell.Cell("F1"), wsRemoveCell.Range("A1:E1").RangeAddress.ToStringRelative(true));
+
+            wsRemoveCell.Cell("F1").Comment.AddText("Test");
+
+            wsRemoveCell.Cell("F1").Delete(XLShiftDeletedCells.ShiftCellsLeft);
+
+            workbook.SaveAs(filePath);
+        }
+    }    
+}
```

Comment:
I'm curious how can you go without the unit tests?? =-)

Reply:
I'm not sure what you mean by this. ;)

Reply:
I mean, you implement the advanced functionality and do not use unit tests for some reason... You do realize that they have to be created anyway, don't you? So why not implement them in advance?

Reply:
I guess because I was unsure my initial attempt at implementing these things was going to be correct so I would just have to rewrite the tests after fixing.